frontend/src/contexts/FirebaseAuth.context.tsx

```diff
@@ -44,6 +51,10 @@ export function FirebaseAuthProvider(props: { children: any }) {
     return deleteUser(user);
   }
 
+  function isAdmin(): boolean {
+    return user.uid === import.meta.env.VITE_ADMIN_UID;
+  }
+
```

Comment:
lol, playing with fire here.
all `VITE_` env variables will be replaced onto the frontend when built, buttttt oh well 
ideally a call made to the backend is done for this